tests/integration/run_tests.py

```diff
@@ -546,12 +546,6 @@ def define_tests():
             f"{run} {demo} --vars N_SAMPLES=2 OUTPUT_PATH=./{OUTPUT_DIR} --local",
             [
                 ReturnCodeCond(),
-                ProvenanceCond(
```

Comment:
why is this test removed?

Reply:
It was searching for a string about the predict script, which was removed to make way for spellbook predict